### ISSUE TITLE
Can't write or modify 'x360ce.ini' file

### ISSUE THREAD

**Issue by toxygenicsubstance** [x360ce/x360ce#703]
After unpacking the x360ce from rar archive i opened it, configured my gamepad saved everything and closed it. Opened the game (Cuphead btw) - the gamepad didn't work. So i opened the app again to see this error. After deleting x360ce.ini i can now open the app again but it appears after i save my settings. I unmarked the "read only" on the whole folder and everything in it - it still appears. Am i doing something wrong?

**Comment by Squall-Leonhart** [comment]
you have placed the app in a UAC protected location.

read only doesn't work on folders, it just changes the state of files inside.

**Comment by xDShot** [comment]
Same here. File is fully accessible. Running from admin.

**Comment by spolezhaev** [comment]
Have the same issue. I shoud delete ini file and dll every time before launching the game. The game is installed in external drive, maybe this is an issue

**Comment by asoltesz** [comment]
I also have this problem under Linux (wine 3.0) 32bit.

Workaround:
- move the x360ce.ini to an other folder (say "backup")
- restart CE (will create a new x360ce.ini)
- when CE starts, the "New device detected" dialog will appear for the controller
- select "Browse my computer for settings"
- select the "backup" folder
- CE will report that there is an ini file in the folder, confirm that you want to load it
- CE loads the settings you have saved
- pronto...you can continue re-editing your controller mappings

**Comment by Squall-Leonhart** [comment]
we don't support wine.

**Comment by nstgc** [comment]
Sure, but why actively prevent us from using this in Wine?

**Comment by Squall-Leonhart** [comment]
lol.

**Comment by idealius** [comment]
I had this problem and it appeared to be caused by the installation of the game being located on an external drive. I resolved it by setting up Xbox360ce on my C drive and pointning to the game location on my external drive (from the game settings tab). Then I saved it, copied the files to the location of the game on my external drive and run the game. Anytime I would want to change a setting the process would have to be repeated, but it works. My external drive is exFAT and I'm running Windows 10. I also set the Xbox360ce.exe to Compatibility Mode for Windows 7 and ran it as administrator from the right-click menu.
